README.md

```diff
@@ -188,31 +187,31 @@ const Component = React.lazy(() => {
 
   let module;
   switch (effectiveType) {
-    case "3g":
-      module = import(/* webpackChunkName: "light" */ "./Light.js");
+    case 'slow-2g':
```

Comment:
I intentionally excluded the other supported values here as they were making the code-snippet look a little on the long side (I imagine most users would use 2 values) 😄 

That said, perhaps a good middle ground is a line after each code-snippet that notes the supported return values? e.g `2g`, `3g` etc. Wdyt?

Reply:
I got you.

So for Network, Save Data, CPU Cores / Hardware Concurrency, Memory, after code snippet, we can add some line that notes the supported return values.

For example for Network, `effectiveConnectionType values can be slow-2g, 2g, 3g, and 4g`.

Am I correct?

Reply:
```
  switch (effectiveType) {
    case '3g':
      module = import(/* webpackChunkName: "light" */ './Light.js');
      break;
    case '4g':
      module = import(/* webpackChunkName: "full" */ './Full.js');
      break;
    default:
      module = import(/* webpackChunkName: "full" */ './Full.js');
      break;
  }
```

I thought that if effective connection type is `2g`, the user might end up downloading `Full.js`.

Reply:
> For example for Network, effectiveConnectionType values can be slow-2g, 2g, 3g, and 4g.

Yes, exactly!